### DISCUSSION TITLE
Change dialog window to browser window.

### DISCUSSION THREAD

**Discussion by Petri1x1** [tinymce/tinymce#9854] [General]
In the old tinymce version 4, dialogs were handled by a new separate browser window / popup window (it was possible to move the window, minimize, maximize, close with a cross). In the new tinymce version 7, only the dialog is now a modal across the screen - you can't minimize the window, for example. Is it possible to override this settings in the new tinymce to a browser window / popup window? By "dialog window" I mean for example the codeeditor or inserting a link.

**Comment by TheSpyder** [comment]
Sorry, no, the dialog style changed in TinyMCE 5 and there's no way to bring back the popup window style.
